src/node/node_state.h

```diff
@@ -596,6 +599,282 @@ namespace ccf
         std::chrono::milliseconds(config.joining.join_timer));
     }
 
+    template <typename T>
+    void send_refresh_jwt_keys(T msg)
+    {
+      auto body = serdes::pack(msg, serdes::Pack::Text);
+
+      http::Request request(fmt::format(
+        "/{}/{}",
+        ccf::get_actor_prefix(ccf::ActorsType::members),
+        "jwt_keys/refresh"));
+      request.set_header(
+        http::headers::CONTENT_TYPE, http::headervalues::contenttype::JSON);
+      request.set_body(&body);
+      // Need a custom authentication policy that accepts only node certs.
+      // See https://github.com/microsoft/CCF/issues/1904
+      // http::sign_request(request, node_sign_kp);
+      auto packed = request.build_request();
+
+      auto node_session = std::make_shared<enclave::SessionContext>(
+        enclave::InvalidSessionId, node_cert.raw());
+      auto ctx = enclave::make_rpc_context(node_session, packed);
+
+      const auto actor_opt = http::extract_actor(*ctx);
+      if (!actor_opt.has_value())
+      {
+        throw std::logic_error("Unable to get actor");
+      }
+
+      const auto actor = rpc_map->resolve(actor_opt.value());
+      auto frontend_opt = this->rpc_map->find(actor);
+      if (!frontend_opt.has_value())
+      {
+        throw std::logic_error(
+          "RpcMap::find returned invalid (empty) frontend");
+      }
+      auto frontend = frontend_opt.value();
+      frontend->process(ctx);
```

Comment:
Check that this has a return value?

Reply:
The endpoint doesn't return anything, why should I check?

Reply:
You should check that the endpoint returned `HTTP_STATUS_OK` at least?

Reply:
The endpoint either returns 200 or 500, and both are expected results.